interface-spec.md

```diff
@@ -117,3 +117,56 @@ Triples always have `.graph` set to DefaultGraph.
 - `DefaultGraph .defaultGraph()` returns an instance of DefaultGraph.
 - `Triple .triple(Term subject, Term predicate, Term object, [Term graph])` returns a new instance of Triple.
 - `Quad .quad(Term subject, Term predicate, Term object, [Term graph])` returns a new instance of Quad.
+
+## Stream interfaces
+
+Streams are used only in a readable manner.
+This requires only a single queue per stream, which simplifies implementations and doesn't have performance drawbacks, compared to writeable streams.
+
+### Stream extends EventEmitter
+
+**Methods:**
+
+- `Quad .read()`
+  This method pulls a quad out of the internal buffer and returns it.
+  If there is no quad available, then it will return null. 
+
+**Events:**
+
+- `readable`
```

Comment:
`readable` is an [official Node.js event](https://nodejs.org/api/stream.html#stream_event_readable). So let's stay with that standard.